content/src/content/jcr_root/apps/core/wcm/components/teaser/v1/teaser/_cq_design_dialog/.content.xml

```diff
@@ -62,6 +62,12 @@
                                         text="Hide title"
                                         name="./titleHidden"
                                         value="{Boolean}true"/>
+                                    <titleType
+                                        jcr:primaryType="nt:unstructured"
```

Comment:
@cjelger to be backward compatible the toggle should be enabled by default, so shouldn't this be set enabled by default with `checked =true` ?

Reply:
@msagolj Please check again, as discussed on Slack, I had to "invert" the condition because it's not possible to set the checkbox to default to `true` ... :-(